llm_bioinformatics_research/src/UserProfile/UserProfile.js

```diff
@@ -191,114 +168,74 @@ const UserProfile = () => {
         </Button>
       </Box>
 
-      {section === 'profile' && (
-        <Box className="profile-section">
-          <Box sx={{ position: 'relative', display: 'inline-block' }} className="avatar-container">
-            <Avatar
-              alt="User Image"
-              src={user.image}
-              sx={{ width: '15vh', height: '15vh', mr: 2 }}
-            />
-            <IconButton
-              sx={{
-                position: 'absolute',
-                bottom: 0,
-                right: 0,
-                bgcolor: 'background.paper',
-                borderRadius: '50%',
-                border: '1px solid',
-                borderColor: 'divider',
-                width: '5vh',
-                height: '5vh',
-              }}
-              component="label"
-            >
-              <EditIcon />
-              <input type="file" hidden accept="image/*" onChange={handleImageChange} />
-            </IconButton>
-          </Box>
-
-          <TextField
-            label="Name"
-            name="name"
-            value={user.name}
-            className="input-field"
-            margin="normal"
-            InputProps={{ readOnly: true }} // Make name field read-only
-          />
-          <TextField
-            label="Email"
-            name="email"
-            value={user.email}
-            className="input-field"
-            margin="normal"
-            InputProps={{ readOnly: true }} // Make email field read-only
+      <Box className="profile-section">
+        <Box sx={{ position: 'relative', display: 'inline-block' }} className="avatar-container">
+          <Avatar
+            alt="User Image"
+            src={user.image}
+            sx={{ width: '15vh', height: '15vh', mr: 2 }}
           />
-          <TextField
-            label="Phone Number"
-            name="phone"
-            value={user.phone}
-            onChange={handleInputChange}
-            className="input-field"
-            margin="normal"
-            error={Boolean(phoneError)} 
-            helperText={phoneError} 
-          />
-          <TextField
-            label="Location"
-            name="location"
-            value={user.location}
-            onChange={handleInputChange}
-            className="input-field"
-            margin="normal"
-            error={Boolean(locationError)} 
-            helperText={locationError} 
-          />
-          <Button variant="contained" color="primary" onClick={handleUpdate}>
-            Update
-          </Button>
+          <IconButton
```

Comment:
I am not sure if this can be done in the scope of this sprint, but I can imagine two more features being added to the profile page:

1) When the user clicks update, we can add a dialogue box that says "Are you sure you want to update this information?" or "Please verify the following changes".

2) If the user attempts to navigate away from the page with partially made changes, we can add a dialogue box that says "Do you want to save your change?"